pkg/plugin/config.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package plugin provides the implementation of the JVS plugin interface.
+package plugin
```

Comment:
Do we need a validate method to validate the config?

Reply:
There isn't a pattern for the github ID and installation ID, and the validation for PEM shouldn't be done here?

I will added a `newPluginConfig` function that read these values from env vars. And `plugin.go` will use this function to create new config and provide to `NewValidator()`. Something like this: https://github.com/abcxyz/jvs-plugin-github-dev/blob/sailorlqh/implement_validator/pkg/plugin/config.go#L31

So I think the validation is not necessary?

Reply:
maybe a followup, we could use the new cli pattern: https://github.com/abcxyz/pmap/blob/main/pkg/server/config.go#L97

Reply:
We should do a similar thing as https://github.com/abcxyz/jvs-plugin-jira/blob/main/pkg/cli/server.go, at which point we should turn the config to be a flag set. Could be a follow up